app/presenters/content_items_presenter.rb

```diff
@@ -1,13 +1,13 @@
 class ContentItemsPresenter
   include Kaminari::Helpers::HelperMethods
-  attr_reader :title, :content_items, :pagination, :search_parameters
+  attr_reader :title, :content_items, :pagination, :filter, :search_parameters
   delegate :page, :total_pages, :prev_link?, :next_link?, :prev_label, :next_label, to: :pagination
+  delegate :document_type_options, :organisation_options, to: :filter
```

Comment:
I feel this is a bit split brain - we have delegation for these options, but not other attributes `document_type`, `organisation_name` etc... Yet we also pass the FilterPresenter to view anyway.  For consistency, we should either delegate everything or nothing at all.